OUDS/Core/Themes/Orange/Sources/_OUDSThemesOrange.docc/OUDSThemesOrange.md

```diff
@@ -239,9 +231,7 @@ struct YourApp: App {
 ### By overriding only existing tokens providers
 
 Quite simple and similar to the previous solution, but give only the providers to the ``OrangeTheme``.
-But beware, in all cases if will crash is some providers are missing.
-Ensure you add **all** the expected providers. You can mix your own providers and those from Orange theme.
-
+But we recommnd tod efine your own theme for clariy reasons.
```

Comment:
typo  But we recommnd tod efine your own theme for clariy reasons.
-> But we recommnd to define your own theme for clarily reasons.